src/main/webapp/src-sw.js

```diff
@@ -0,0 +1,8 @@
+workbox.routing.registerRoute(
```

Comment:
take a look at this https://developers.google.com/web/tools/workbox/modules/workbox-routing
you can use regex here, so there is no need to specify the host
I think we can use cacheFirst for js and css files, I'm not sue if we even need this rule with networkFirst